packages/postinstall-node-version-check/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@aws-sdk/postinstall-node-version-check",
+  "version": "3.0.0",
+  "scripts": {
```

Comment:
`description`?

Reply:
Added in [`fcd0261` (#2585)](https://github.com/aws/aws-sdk-js-v3/pull/2585/commits/fcd0261674346ad0ea4a2ce0fde62ef255668995)